src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationBraceAdjustment.java

```diff
@@ -15,17 +15,26 @@
  * @author  jrichard                                                          //indent:1 exp:1
  */                                                                           //indent:1 exp:1
 public class InputIndentationBraceAdjustment //indent:0 exp:0
-  { //indent:2 exp:2
+{ //indent:0 exp:2 warn
```

Comment:
declared config is `braceAdjustment = 2` 
please restore this line, lets keep it as it was before as much as possible.

if you need extra text case - please create new file.
Existing text cases should be changes only if there was bug in them or behavior is changing (change in comment only).

Reply:
How bad it is that I always forget the same thing.

Reply:
No problems, just keep fixing it, one day it become a habit.